src/__tests__/nav.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, fireEvent, render, screen} from '@testing-library/react';
+import Toggle from '../components/CustomInputs/Toggle';
+import Nav from '../components/Nav';
+
+describe('Nav tests', () => {
+  afterEach(cleanup);
+
+  describe('Ensure components get rendered', () => {
+    it('Nav Header Wrapper', () => {
+      render(<Nav />);
+      expect(screen.getByTestId('homepage')).toBeInTheDocument();
```

Comment:
`getByTestId` should be used sparingly, look if you can use more semantic HTML elements for your components and use `getByRole` queries, instead.